docs/markdown-output-specification.md

```
@@ -0,0 +1,307 @@
+- Optional timestamp separated by em dash (`—`)
+- Colon (`:`) when no timestamp present
+- All markdown special characters escaped in speaker names and timestamps
+
+### Content Types
+
+#### 1. Text Content
+```markdown
+**User:**
+Hello *world* with _emphasis_!
+```
+- Markdown special characters escaped: `\ ` * _ { } [ ] ( ) # + - . ! |`
```

markdownlint-cli2 (0.23.1)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)